pmd-java/src/main/resources/category/java/codestyle.xml

```diff
@@ -1761,22 +1761,26 @@ public class Foo {
         </example>
     </rule>
 
-    <rule name="UnnecessaryLocalBeforeReturn"
+    <rule name="UnnecessaryLocalBeforeBranch"
           language="java"
-          since="3.3"
-          message="Consider simply returning the value vs storing it in local variable ''{0}''"
-          class="net.sourceforge.pmd.lang.java.rule.codestyle.UnnecessaryLocalBeforeReturnRule"
-          externalInfoUrl="${pmd.website.baseurl}/pmd_rules_java_codestyle.html#unnecessarylocalbeforereturn">
+          since="7.15.0"
+          message="Alternate coupling burden on immediately following end ''{0}''."
+          class="net.sourceforge.pmd.lang.java.rule.codestyle.UnnecessaryLocalBeforeBranchRule"
+          externalInfoUrl="${pmd.website.baseurl}/pmd_rules_java_codestyle.html#unnecessarylocalbeforebranch">
         <description>
-Avoid the creation of unnecessary local variables
+Alternate coupling burden on immediately following end.
```

Comment:
Sorry, I don't understand this sentence. The old sentence was more understandable, although it didn't provide an explanation, why unnecessary local variables are "bad".